payments-core/src/main/java/com/stripe/android/payments/bankaccount/ui/CollectBankAccountActivity.kt

```diff
@@ -27,7 +32,7 @@ class CollectBankAccountActivity : AppCompatActivity() {
         CollectBankAccountContract.Args.fromIntent(intent)
     }
 
-    private lateinit var financialConnectionsPaymentsProxy: FinancialConnectionsPaymentsProxy
+    private lateinit var financialConnectionsLauncher: FinancialConnectionsSheetLauncher
```

Comment:
We don't need to safely access FC classes anymore since `financial-connections-core` (that just contains relevant models) is pulled transitively. We can directly build the launcher now.